dns_infoblox.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env sh
+
+dns_infoblox_add() {
+
+  fulldomain=$1
+  txtvalue=$2
+
+  baseurlnObject="https://$Infoblox_Server/wapi/v2.2.2/record:txt?name=$fulldomain&text=$txtvalue"
+
+  _info "Using Infoblox API"
+
+  _debug fulldomain "$fulldomain"
+
+  _debug txtvalue "$txtvalue"
+
+  #_err "Not implemented!"
+
+  if [ -z "$Infoblox_Creds" ] || [ -z "$Infoblox_Server" ]; then
+    Infoblox_Creds=""
+    Infoblox_Server=""
+    _err "You didn't specify the credentials or server yet (Infoblox_Creds and Infoblox_Server)."
+    _err "Please set them via EXPORT ([username:password] and [ip or hostname]) and try again."
+    return 1
+  fi
+
+  #save the login info to the account conf file.
+  _saveaccountconf Infoblox_Creds "$Infoblox_Creds"
+  _saveaccountconf Infoblox_Server "$Infoblox_Server"
+
+  result=$(curl -k -u $Infoblox_Creds -X POST $baseurlnObject)
```

Comment:
do not use `curl` or `wget` in your code.
Please use `_post()` function to do post request.